<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>David Hud√°k</author>
 <comment>File contains generated design of main window</comment>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButton_8">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove input</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Add output</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Code</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBox">
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="3" colspan="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="9" column="3" colspan="2">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Editor</string>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="12" column="4">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Add input</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="13">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListView" name="listView">
          <property name="maximumSize">
           <size>
            <width>269</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>All is working properly</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Create New Block</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Compile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Clean Board</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>Add Block</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>Zoom +</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>Zoom -</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuComposite_Blocks">
    <property name="title">
     <string>Composite Blocks</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuComposite_Blocks"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
